setup.py

```diff
@@ -13,7 +13,7 @@
 min_python_version = "3.6"
 min_numpy_build_version = "1.11"
 min_numpy_run_version = "1.15"
-min_llvmlite_version = "0.33"
+min_llvmlite_version = "0.34.0.dev0"
 max_llvmlite_version = "0.35"
```

Comment:
I think we will need to bump this as well. because of `'llvmlite >={},<{}'.format(min_llvmlite_version, max_llvmlite_version),`